docs/content/howto/short-lived-entities.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Log short lived data
+order: 5
+description: How to log data that isn't valid for the whole recording
+---
+In order to create coherent views of streaming data, the Rerun Viewer shows the latest values for each visible entity at the current timepoint. But some data may not be valid for the entire recoding even if there are no updated values. How do you tell Rerun that something you've logged should no longer be shown?
+
+## Log entities as cleared
+The most straight forward option is to log that an entity has been cleared. For example, if you have an object tracking application, you're code might look something like this:
+```python
+...
+for frame in sensors.read():
+    # Associate the following logs with `frame == frame.id`
+    rr.set_time_sequence("frame", frame.id)
+    # Do the actual tracking update
+    tracker.update(frame)
+    if tracker.is_lost:
+        # Clear everything on or below `tracked/{tracker.id}`
+        # and that happened on or before `frame == frame.id`
+        rr.log(f"tracked/{tracker.id}", rr.ClearEntity(recursive=True))
+    else:
+        # Log data to the main entity and a child entity
+        rr.log(f"tracked/{tracker.id}", rr.Rect2D(tracker.bounds))
+        rr.log(f"tracked/{tracker.id}/cm", rr.Point2D(tracker.cm))
+
+```
+## Clarify data meaning
+In some cases, the best approach may be to rethink how you log data to better express what is actually happening. Take the following example where update frequencies don't match:
+
+```python
+...
+for frame in sensors.read():
+    # Associate the following logs with `frame = frame.id`
+    rr.set_time_sequence("frame", frame.id)
+    # Log every image that comes in
+    rr.log("input/image", rr.Image(frame.image))
+    if frame.id % 10 == 0:
+        # Run detection every 10 frames
+        detection = detector.detect(frame)
+
+        # Woops! These detections will not update at the
+        # same frequency as the input data and thus look strange
+        rr.log("input/detections", rr.Rect2D(detection.bounds))
+```
+You could fix this example by logging `rr.ClearEntity`, but in this case it makes more sense to change what you log to better express what is happening. Here is an example fix:
```

Comment:
```suggestion
You could fix this example by logging `rr.ClearEntity`, but in this case it makes more sense to change what you log to better express what is happening. Re-logging the image to another namespace on only the frames where the detection runs makes it explicit which frame was used as the input to the detector. This will create a second view in the viewer that always allows you to see the frame that was used for the current detection input.

Here is an example fix:
```